frontend/src/components/search/SearchFilterAccordion/SearchFilterSection/SearchFilterSection.tsx

```diff
@@ -55,6 +57,9 @@ const SearchFilterSection: React.FC<SearchFilterSectionProps> = ({
     }
   }, [option.children]);
 
+  const getHiddenName = (name: string) =>
+    accordionTitle === "Agency" ? `agency-${name}` : name;
```

Comment:
Should we do this regardless of the `accordionTitle`, and prefix the `name` with whatever the `accordionTitle` is? Might need to transform uppercase letters to lowercase and spaces to dashes. It'd only apply to the agency filter, as it's the only filter using sections, but the code might be more flexible. 

(Thinking about if UI strings change. I suppose this could still break.)

Reply:
For reference here's what gets passed when we try to update results:

```
    { name: 'query', value: 'asdf' },
    { name: 'status-archived', value: 'on' },
    { name: 'funding-instrument-cooperative_agreement', value: '' },
    { name: 'agency-US-ABC', value: '' },
    { name: 'search-sort-by', value: 'opportunityTitleDesc' },
    { name: 'currentPage', value: '3' }
    
    ```